dashboard/test/controllers/api_controller_queries_test.rb

```diff
@@ -23,7 +23,7 @@ def setup
 
     sign_in_as section.teacher
 
-    assert_queries 11 do
+    assert_queries 15 do
```

Comment:
Are we worried about all of these query counts going up by so much? Do we incur this cost on every request or just login?

Reply:
These requests are made in `after_set_user`, so it should just be when the user logs in. It follows the pattern for setting cookies at login like the experiments cookie. It's not ideal that I've added extra queries, but I didn't see a great alternative.

Reply:
I moved where the cookies get set to the homepage which eliminated the extra queries.